setup.py

```diff
@@ -0,0 +1,22 @@
+import sys
+
+from setuptools import setup
+
+__author__ = "Eric Yvorchuk"
+__email__ = "eyvorchuk@uvic.ca"
+__version__ = "0.1.0"
+
+setup(
+    name="wps_tools",
+    description="A collection of modules used to help create wps processes",
+    keywords="wps pywps wps_tools",
+    version=__version__,
+    url="https://github.com/pacificclimate/wps-tools",
+    author=__author__,
+    author_email=__email__,
+    packages=["wps_tools"],
```

Comment:
I _think_ it is better to use `find_packages()` but I don't know for sure. So we can leave this as is for now.